test/words/test/TestINodeAssign.java

```diff
@@ -0,0 +1,36 @@
+package words.test;
+import static org.junit.Assert.*;
+import words.ast.*;
+import words.exceptions.*;
+import words.environment.*;
+
+import org.junit.Test;
+
+public class TestINodeAssign extends TestINode {
+	
+	@Test
+	public void assignProperty() throws WordsRuntimeException {
```

Comment:
We should also have a test that this WordsRunTimeException is thrown when it should be thrown.